From: Matus Valo <matusvalo@gmail.com>
Date: Thu, 3 Jan 2019 06:43:31 -0800
Subject: Fix failing unittests of pyamqp transport.

Failing unittests were caused by commit f16df2a17630c9804a6da614443c5e862271823f in pyamqp.
---
 t/unit/transport/test_pyamqp.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/unit/transport/test_pyamqp.py b/t/unit/transport/test_pyamqp.py
index 8292991..32094b4 100644
--- a/t/unit/transport/test_pyamqp.py
+++ b/t/unit/transport/test_pyamqp.py
@@ -68,11 +68,11 @@ class test_Channel:
         assert self.channel.connection is None
 
     def test_basic_consume_registers_ack_status(self):
-        self.channel.wait_returns = 'my-consumer-tag'
+        self.channel.wait_returns = ['my-consumer-tag']
         self.channel.basic_consume('foo', no_ack=True)
         assert 'my-consumer-tag' in self.channel.no_ack_consumers
 
-        self.channel.wait_returns = 'other-consumer-tag'
+        self.channel.wait_returns = ['other-consumer-tag']
         self.channel.basic_consume('bar', no_ack=False)
         assert 'other-consumer-tag' not in self.channel.no_ack_consumers
 
